man/fi_endpoint.3.md

```diff
@@ -616,19 +616,20 @@ if data ordering is not maintained, message ordering may be.
 : Read after write size.  If set, an RMA or atomic read operation
   issued after an RMA or atomic write operation, both of which are
   smaller than the size, will be ordered.  The RMA or atomic read
-  operation will see the results of the previous RMA or atomic write.
+  operation will see the results of the previous RMA or atomic write
+  at the target memory location.
```

Comment:
OK, my intent was that the reader should not be confused that the *entire* memory region is updated atomically, before the second operation starts. e.g. I don't want someone to think that the entire RMA or atomic read will see the value of the previous entire RMA or atomic write.

I was concerned about misinterpretation since 'memory region' implies the entire range registered in libfabric speak.